<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShadersEditor</class>
 <widget class="QWidget" name="ShadersEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="layoutToolBar"/>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QDockWidget" name="dockVertex">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="accessibleName">
          <string>dockVertex</string>
         </property>
         <property name="features">
          <set>QDockWidget::NoDockWidgetFeatures</set>
         </property>
         <property name="windowTitle">
          <string>Vertex Shader</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents_2">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="layoutVertex"/>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QDockWidget" name="dockFragment">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="accessibleName">
          <string>dockFragment</string>
         </property>
         <property name="features">
          <set>QDockWidget::NoDockWidgetFeatures</set>
         </property>
         <property name="windowTitle">
          <string>Fragment Shader</string>
         </property>
         <widget class="QWidget" name="dockWidgetContents">
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="layoutFragment"/>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actNew">
   <property name="icon">
    <iconset resource="MagicGameEditor.qrc">
     <normaloff>:/MagicGameEditor/Images/new.png</normaloff>:/MagicGameEditor/Images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New Shader</string>
   </property>
  </action>
  <action name="actCompile">
   <property name="icon">
    <iconset resource="MagicGameEditor.qrc">
     <normaloff>:/MagicGameEditor/Images/color_wheel.png</normaloff>:/MagicGameEditor/Images/color_wheel.png</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="toolTip">
    <string>Compile Shader</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="icon">
    <iconset resource="MagicGameEditor.qrc">
     <normaloff>:/MagicGameEditor/Images/delete.png</normaloff>:/MagicGameEditor/Images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete Shader</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MagicGameEditor.qrc"/>
 </resources>
 <connections/>
</ui>
